sources/packages/backend/apps/api/src/utilities/string-utils.ts

```diff
@@ -15,3 +15,13 @@ export const removeWhiteSpaces = (stringValue: string): string => {
 export const removeDoubleWhiteSpaces = (stringValue: string): string => {
   return stringValue.replace(/\s+/g, " ");
 };
+
+/**
+ * Encodes a filename for Content-Disposition header
+ * @param fileName Original filename to encode
+ * @returns Object containing encoded filename
+ */
+export function encodeFileNameForContentDisposition(fileName: string): string {
+  const encodedFileName = encodeURIComponent(fileName);
```

Comment:
As we are just using `encodeURIComponent` please remove the util as the purpose seems to be more linear.